python/cudf/cudf/core/column/string.py

```diff
@@ -3612,6 +3615,56 @@ def translate(self, table, **kwargs):
             cpp_translate(self._column, table), **kwargs
         )
 
+    def filter_characters(self, table, keep=True, repl=None, **kwargs):
+        """
+        Remove characters from each string using the character ranges
+        in the given mapping table.
+
+        Parameters
+        ----------
+        table : dict
+            This table is a range of Unicode ordinals to filter.
+            The minimum value is the key and the maximum value is the value.
+            You can use `str.maketrans()
+            <https://docs.python.org/3/library/stdtypes.html#str.maketrans>`_
+            as a helper function for making the filter table.
+            Overlapping ranges will cause undefined results.
+            Range values are inclusive.
+        keep : boolean
+            If False, the character ranges in the ``table`` are removed.
+            If True, the character ranges in the ``table`` are removed.
```

Comment:
This might need to be changed as they are the same.

Reply:
```suggestion
            If False, the character ranges in the ``table`` are removed.
            If True, the character ranges in the ``table`` are kept.
```

Reply:
Could catch Vibhu.